Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 /// A structure that describes a Nostr event.
 ///
 /// > Note: [NIP-01 Specification](https://github.com/nostr-protocol/nips/blob/master/01.md#events-and-signatures)
-public class NostrEvent: Codable, Equatable, Hashable, ContentWarningTagInterpreting, LabelTagInterpreting {
+public class NostrEvent: Codable, Equatable, Hashable, AlternativeSummaryTagInterpreting, ContentWarningTagInterpreting, ExpirationTagInterpreting, LabelTagInterpreting {
```

Comment:
Making this change allows us to keep the base NostrEvent class clean for only the NIP-01 implementation. Any other NIPs that apply to all event kinds can be introduced as a separate protocol/extension that NostrEvent extends from. It'll let us maintain the code much easier.